src/game/game.h

```diff
@@ -1683,6 +1683,8 @@ namespace hud
     extern int raceinventory(int x, int y, int s, float blend);
     extern int drawscore(int x, int y, int s, int m, float blend, int count);
     extern void cleanup();
+    extern char const* system_time;
+    extern bool show_system_time();
```

Comment:
You should be able to get away without some global static variable. This one is particularly annoying because it's polluting _all_ files' namespace.